server/src/test/java/com/tisza/tarock/game/phase/util/ExtendedBotPlayer.java

```diff
@@ -0,0 +1,34 @@
+package com.tisza.tarock.game.phase.util;
+
+import com.tisza.tarock.message.*;
+import com.tisza.tarock.server.database.*;
+import com.tisza.tarock.server.player.bot.*;
+
+import java.util.*;
+
+public class ExtendedBotPlayer extends BotPlayer
+{
```

Comment:
Let's move this class to com.tisza.tarock.server.player.bot too. And maybe it could be named TestBotPlayer.